rdma-ndd/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# COPYRIGHT (c) 2016 Intel Corporation. See COPYING file
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${NO_STRICT_ALIASING_FLAGS}")
+
```

Comment:
I'll check.  Honestly I copied that from the other CMakeLists and did not really think about it.